src/VCS/Adapter/Git/GitHub.php

```diff
@@ -143,7 +144,7 @@ public function getRepositoryName(string $repositoryId): string
         $response = $this->call(self::METHOD_GET, $url, ['Authorization' => "Bearer $this->accessToken"]);
 
         if (!isset($response['body']['name'])) {
-            throw new Exception("Repository name not found");
+            throw new RepositoryNotFound("Repository not found");
```

Comment:
```suggestion
            throw new Exception("Repository not found");
```

You can use the VCS\Exception class directly

Reply:
I did it this way so that in Appwrite I can do `try/catch`, but only catch `RepositoryNotFound` exception and throw exception accordingly there.